src/servers/http_server.cc

```diff
@@ -726,6 +726,21 @@ HTTPAPIServer::EVBufferToInput(
             .c_str());
   }
 
+  // Initialize System Memory for Output if it uses shared memory
```

Comment:
I think the changes to ensembler, provider or trtserver can be much simplied... we should discuss.

Reply:
Sure